samples/BoltOn.Samples.Infrastructure/Data/CollegeDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using BoltOn.Data.CosmosDb;
+using Microsoft.Extensions.Options;
+
+namespace BoltOn.Samples.Infrastructure.Data
+{
+    public class CollegeDbContext : BaseCosmosDbContext
+    {
+        //private readonly CosmosSettings _settings;
+        private const string DatabaseName = "College";
+
+        public CollegeDbContext(IOptions<CosmosDbSettings> settings) : base(settings.Value, DatabaseName)
```

Comment:
pass the `IOptions<CosmosDbSettings>` as it's to the base class, and get the settings.Value inside the base class